src/components/FooterContainer/__tests__/FooterContainer.test.tsx

```diff
@@ -17,12 +17,23 @@ describe('<Footer />', () => {
 
   it('should render correctly', () => {
     configuration.links.privacy = 'https://www.amsterdam.nl/privacy/'
+    configuration.links.about = 'https://www.amsterdam.nl/overdezesite/'
+    configuration.links.accessibility = '/toegankelijkheid/'
+
     const { container, getByTestId } = render(withAppContext(<Footer />))
 
     expect(container.querySelector('div.no-print')).toBeInTheDocument()
     expect(getByTestId('disclaimer')).toBeInTheDocument()
     expect(
       container.querySelector('a[href="https://www.amsterdam.nl/privacy/"]')
     ).toBeInTheDocument()
+    expect(
+      container.querySelector(
+        'a[href="https://www.amsterdam.nl/overdezesite/"]'
+      )
+    ).toBeInTheDocument()
+    expect(
+      container.querySelector('a[href="/toegankelijkheid/"]')
+    ).toBeInTheDocument()
```

Comment:
Best to reference the configuration settings to prevent coincidence.
Like this:
```javascript
expect(
  container.querySelector(
    `a[href="${configuration.links.about}"]`
  )
).toBeInTheDocument()
```
Event better would be to use `screen`:
```javascript
import { render, screen } from '@testing-library/react'

expect(
  screen.getByRole('link', { href: configuration.links.about })toBeInTheDocument()
)
```

Reply:
👍

Reply:
@janjaap als je getByRole gebruikt dan kun je als option alleen 'name' meegeven.

Reply:
I see. Maybe this:
```javascript
expect(screen.getByRole('link')).toHaveAttribute('href', configuration.links.about)
```